### PR TITLE
Portfolio works section done

### PR DESCRIPTION
**Pull request by NicolasAndrehh**
# Milestone - Works section completed
In this pull request I implement the following features:
- Added `works cards` with his respective information
- Added `about me` section
- Added my resume section where I put `Languajes`, `Frameworks` and `Skills` that I been working with
- Added a simple `form`

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -95,11 +108,180 @@
...
+    text-decoration: none;
+    background-color: #36b37e;
+    padding: 0.75rem;
+    font-size: 1rem;
+    line-height: 1.5rem;
+    color: #fff;
+    letter-spacing: 0.03em;
+    text-align: center;
+    margin: 0.75rem auto;
+  }
```
Comment (TadesseDev) [inline styles.css:186]: - [ ] Dear @NicolasAndrehh, Please make sure to follow the button styling: `Active`, `Hover `, and `Pressed` for all your buttons. I can see you have a hover effect but it's not enough and not according to the Figma design. This is to make sure your design is identical to the figma design.

Screenshot 2023-01-13 092215

File: index.html [hunk 2]
```diff
@@ -50,6 +50,185 @@ <h1>Hey there. I'm Nicolas<br><span>I'm a software developer</span></h1>
...
+            </div>
+        </section>
+
+        <section id="aboutme-section">
+            <h2>About me</h2>           
+            <img src="images/Mask Group.png" alt="vector" class="vector">
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+            <a href="#" class="button">Get My Resume</a>
+
+            <div class="aboutme-card">
```
Comment (TadesseDev) [inline index.html]: ###### Great job adding the decoration, But I need to mention two points here. 
- [ ] One: your first decoration image is not a background image, remember decorations are always backgrounded images, not `img` tags. 
- [ ] Two: your second image is expected to stick to the `About Me` content, as you can see from the screenshot I added it moves down to the other contents. Please address this for a better layout. 

Screenshot 2023-01-13 202937

File: index.html [hunk 3]
```diff
@@ -50,6 +50,185 @@ <h1>Hey there. I'm Nicolas<br><span>I'm a software developer</span></h1>
...
+                <img src="images/icon-languages.png" alt="icon-languajes">
+                <h4>Languajes</h4>
+                <ul>
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>Html</li>
+                    <li>CSS</li>
+                </ul>
+            </div>
```
Comment (TadesseDev) [inline index.html]: - [ ] Its awesome that you make technology in each category(e.g., "JavasScript", "HTML" and "CSS") a `ul` with each being inside `li`, But we also need to follow the same structure for technology categories ("Languages", "Frameworks" and "Skills"). Please make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @NicolasAndrehh 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7

## HighlighAshley[ ] Good job overall following the template design ✅
 - [ ]  Correct GitHub Flow  ✅
 - [ ]  Good commit history  ✅
 - [ ]  Good PR title/description  ✅
 - [ ]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (NicolasAndrehh) [reply index.html]: @Tadesse-Alemayehu you mean the whole card of each category should be in a li tag and the three categories should be inside an ul tag?

**Comment by NicolasAndrehh** [comment]
Thank you for your feedback I'm doing all the changes that you suggest!

File: index.html (hunk 3 above)
Comment (TadesseDev) [reply index.html]: Yes @NicolasAndrehh.

**Comment by TadesseDev** [comment]
good job @NicolasAndrehh

**Comment by NicolasAndrehh** [comment]
# Hey @Tadesse-Alemayehu!
I already do all the changes that you suggested, do you think it's well done?

**Review by bienvenuushindi — APPROVED** [review]
Hi @NicolasAndrehh    , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**